CHANGELOG.md

```diff
@@ -20,11 +20,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The web UI now shows runs by date, with the most recent run at the top.
 - The web UI now shows steps in a color-coded way.
 - The `--include-package` flag now also accepts paths instead of module names.
+- Tango is now better at guessing when a step has died and should be re-run.
```

Comment:
This needs to be moved up to "Unreleased"

Reply:
dcdaad0293cdea55b143e27a79fca2e4a9d28934